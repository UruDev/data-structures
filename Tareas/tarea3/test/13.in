# es_AVL, filtrado
insertar_en_binario (12,m)
insertar_en_binario (8,g)
insertar_en_binario (9,j)
imprimir_binario
es_AVL
insertar_en_binario (2,c)
insertar_en_binario (1,a)
insertar_en_binario (4,e)
insertar_en_binario (5,h)
imprimir_binario
es_AVL
insertar_en_binario (7,k)
insertar_en_binario (14,p)
insertar_en_binario (18,s)
imprimir_binario
es_AVL
insertar_en_binario (1,r)
insertar_en_binario (20,z)
insertar_en_binario (2,n)
imprimir_binario
es_AVL
filtrado 3
filtrado 5
filtrado 8
Fin
