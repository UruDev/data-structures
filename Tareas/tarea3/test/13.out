1>#  es_AVL, filtrado.
2>Insertado.
3>Insertado.
4>Insertado.
5>
(12,m)
--(9,j)
-(8,g)
6>No AVL.
7>Insertado.
8>Insertado.
9>Insertado.
10>Insertado.
11>
(12,m)
--(9,j)
---(5,h)
-(8,g)
---(4,e)
--(2,c)
---(1,a)
12>No AVL.
13>Insertado.
14>Insertado.
15>Insertado.
16>
--(18,s)
-(14,p)
(12,m)
---(7,k)
--(9,j)
---(5,h)
-(8,g)
---(4,e)
--(2,c)
---(1,a)
17>AVL.
18>Insertado.
19>Insertado.
20>Insertado.
21>
---(20,z)
--(18,s)
---(1,r)
-(14,p)
--(2,n)
(12,m)
---(7,k)
--(9,j)
---(5,h)
-(8,g)
---(4,e)
--(2,c)
---(1,a)
22>AVL.
23>
--(1,r)
-(2,n)
(2,c)
-(1,a)
24>
--(1,r)
-(2,n)
(4,e)
-(2,c)
--(1,a)
25>
--(1,r)
-(2,n)
(7,k)
--(5,h)
-(4,e)
--(2,c)
---(1,a)
26>Fin.
